### DISCUSSION TITLE
Does Intel Arc work (as a second GPU?)

### DISCUSSION THREAD

**Discussion by Wordsmith9091** [LizardByte/community#159] [Sunshine] [answered]
Thank you for all you're doing for this project.

I'm trying to use an RTX 3080 as my main GPU for gaming, but an Arc a380 for Sunshine encoding, to see how it compares to encoding via the 3080 itself. 

I've selected Quicksync instead of auto as the forced encoder under the advanced tab, but I'm not seeing anything in the output from the Sunshine frontend app about it finding the QuickTime encoder, and there's no sign in task manager of that GPU being used.

Is what I'm trying to do possible, and if so, what am I doing wrong?

**Comment by cgutman** [comment, marked as answer]
The Desktop Duplication API only supports capturing from the GPU used for display. Since cross-device sharing of textures is not supported, we would have to read back each frame from the display GPU to the CPU and then write it to the encoding GPU to encode it. Though it's definitely possible to do, we don't currently support that today (and performance may be worse due to all the extra copies of each frame). Since NVENC is special dedicated encoding silicon that requires little/no involvement of any of the compute cores used for games, it's unlikely to have a noticeable performance impact on your games, if that's what you're wondering about.

But if you want to try it anyway, you can connect your display to the A380 and restart Sunshine. In the Graphics Options settings page in Windows, you can pick which GPU you want each game to run on to ensure they still render on the RTX 3080.

**Reply by Wordsmith9091** [reply]
Thanks so much for this explanation. I didn't realize Windows had the option to render on one GPU white outputting display through another. I'll give that a try.

I've found that nvenc on my Rtx 3080, whether on official GameStream or Sunshine, seems to have trouble keeping up with 4K120, even on a Moonlight client (like my Macbook Pro) that should probably be able to handle it, and even at lower bitrates (obviously not all games can hit that locally, too, but for those that do). I get to about 4K70-80. Using Sunshine .18.1 with HDR, even at lower resolutions, I'm also seeing pretty major frame drops during scenes with fast action -- I'm not quite sure if that's because of nvenc or something else. When playing locally with HDR, I can do 4K with HDR and my framerates usually stay pretty high, sometimes hitting the cap I've set of 120.

So I figured I'd experiment with the Arc video encoder and see how it compares.
